include/ultra64.h

```diff
@@ -32,4 +32,183 @@
 #include "ultra64/R4300.h"
 #include "ultra64/ucode.h"
 
+union uObjBg;
+
+void __osPiCreateAccessQueue(void);
+void __osPiGetAccess(void);
+void __osPiRelAccess(void);
+s32 osSendMesg(OSMesgQueue* mq, OSMesg msg, s32 flag);
+void osStopThread(OSThread* thread);
+void osViExtendVStart(u32 value);
+s32 osRecvMesg(OSMesgQueue* mq, OSMesg* msg, s32 flag);
+#if PLATFORM_N64
+void osInitialize(void);
+#else
+#define osInitialize()           \
+{                                \
+    __osInitialize_common();     \
+    __osInitialize_autodetect(); \
+}
+#endif
+void __osInitialize_common(void);
```

Comment:
\n police would like this ifdeffed stuff separated from regular one line declarations, but not a big deal